sycl/test-e2e/Basic/built-ins/scalar_relational.cpp

```diff
@@ -0,0 +1,59 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+// RUN: %if preview-breaking-changes-supported %{ %{build} -fpreview-breaking-changes -o %t_preview.out %}
+// RUN: %if preview-breaking-changes-supported %{ %{run} %t_preview.out%}
+
```

Comment:
nit: Perhaps you could add a comment here to state the objective(s) of this test case.

Reply:
It's not different from existing `marray_relational.cpp` and `vec_relational.cpp` and it never caused troubles to understand what they are doing... Are you asking about preview specifically? If yes, then it's a common idiom and we don't expect to document it in every file that uses it.

Reply:
Nope, I was talking about providing a broader overview of this test case.  To my inexperienced eyes, it wasn't readily clear.